bentoml/_internal/io_descriptors/json.py

```diff
@@ -135,16 +113,6 @@ def __init__(
         json_encoder: t.Type[json.JSONEncoder] = DefaultJsonEncoder,
     ):
         if pydantic_model is not None:
-            if pydantic is None:  # pragma: no cover
```

Comment:
can we add this check back?  this warns the user about missing dependency immediately when the API server boot up, not until it receives a request

Reply:
done